apps/browser/src/autofill/fido2/content/fido2-page-script.ts

```diff
@@ -4,6 +4,8 @@ import { MessageType } from "./messaging/message";
 import { Messenger } from "./messaging/messenger";
 
 (function (globalContext) {
+  document.currentScript.parentNode.removeChild(document.currentScript);
```

Comment:
```suggestion
  if (globalContext.document.currentScript) {
    globalContext.document.currentScript.parentNode.removeChild(
      globalContext.document.currentScript,
    );
  }
```

Took a second look at this, so this implementation works when we are dealing with a DOM injection, as is done with a mv2-based extension.

It breaks the implementation on mv3 though, as no DOM node is injected with that script injection process. Adding this check is required for this to function as expected in both mv3 and mv2.